### DISCUSSION TITLE
d4fc76e9-0802-48bd-83d7-4e068a19d33b

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9571] [Rules Discussions]
# d4fc76e9-0802-48bd-83d7-4e068a19d33b

Secret Ingredients to Quality Software | SSW Rules provides best practices for developing secure, reliable, and efficient .NET, Azure, CRM, Angular, React, Dynamics, and AI applications. Learn more today!

https://www.ssw.com.au/rules/the-best-dependency-injection-container/

**Comment by Shabrych** [comment]
Unity is great product but development has stopped. Dont suppurted version NET greater then NET Core 3.1
﻿Unfortunately, I think I need to add a comment "not recommended" too.
